goon/browserassets/js/browserOutput.js

```diff
@@ -154,6 +155,13 @@ function linkify_fallback(text) {
 	});
 }
 
+function emojify(node) {
+	var rex = /:[\w\d\-_]+:/g;
+	node.innerHTML = node.innerHTML.replace(rex, function ($0) {
```

Comment:
Use a proper argument instead of `$0` like `match`

Reply:
Like this? https://imgur.com/3YJjwhK

Reply:
yeah thats fine, `$0` had no meaning and make it look harder than it is.